config/fixes/convention-eccodes.yaml

```diff
@@ -74,6 +77,7 @@ convention_name:
           - avg_lcc
           - avg_lcc_frac
           - LCC # AQUA ERA5
+          - cll # cmor
```

Comment:
uhm, this does not look like a cmor standard https://github.com/search?q=repo%3APCMDI%2Fcmip6-cmor-tables%20cll&type=code

Reply:
You are right! This is a EC-Earth4 invention .... what should we do about these? Maybe NOT include them and leave them in their separate fixes file ?

Reply:
They are really not very standard and may change in future versions